analysis/deep2_r23_o32/plotting/te_metal_plots.py

```diff
@@ -79,37 +79,47 @@ def plotting_te_metal(fitspath, fitspath_ini, revised=False):
             -> refers to if using the bin_derived_prop_revised temperature
                      and metallicity measurements which right now implement dust attenuation
     """
-    indv_all_file = join(fitspath, filename_dict['indv_bin_info'])
+    if raw:
+        mc = '.MC'
+    else:
+        mc = ''
```

Comment:
It would be better if we use a `suffix` variable, initialize it with `suffix = ''` and then append based on these boolean. Then you just need to use `suffix` for the filenames

Reply:
Also, this is very confusing. `raw` implies no randomization yet it is indicating a suffix of ".MC" here when `raw=True`

Reply:
Fixed